recipes/dask-searchcv/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dask-searchcv" %}
+{% set version = "0.0.1" %}
+{% set md5 = "7d8fb38ba01d7518e2d22e5933f1e419" %}
```

Comment:
It [might be preferable to use sha256](https://github.com/conda-forge/staged-recipes/blob/master/README.md#2-how-do-i-populate-the-hash-field) if possible instead of md5.